Metainfo readme en/de (#4432)
redaxo/redaxo: redaxo/src/addons/metainfo/README.md

> # Meta Infos AddOn

Reviewer: ```suggestion
# Meta Infos Add-On
```

Ich glaube, für Englisch hatten wir uns auf die Schreibweise mit Bindestrich verständigt. Bin aber nicht mehr sicher.

> With the Meta Infos-AddOn the meta fields for articles, categories, media and languages can be created and designed.  

Reviewer: ```suggestion
With the Meta Infos add-on the meta fields for articles, categories, media and languages can be created and designed.

```

> Meta Infos are intended to manage page or media specific information, such as search engine keywords, an individual header image, whether it should be considered as a news teaser, whether it should appear in certain navigations, copyright info for certain photos, etc. 

Reviewer: ```suggestion
Meta Infos are intended to manage page or media specific information, such as search engine keywords, an individual header image, whether it should be considered as a news teaser, whether it should appear in certain navigations, copyright info for certain photos, etc.

```

Metainfo fields can be defined for articles, categories as well as media. 

> Available field types are text, textarea, select, radio, checkbox and more.

Reviewer: ```suggestion
Available field types are `text`, `textarea`, `select`, `radio`, `checkbox` and more.
```

> # Meta Infos Add-On

Reviewer 2: Im Englischen ist es mit den Leerstellen so korrekt, denke ich.
Aber gerade da stellt sich mir auch die Frage mit dem Plural, war da nicht was mit "information" ohne Plural im Englischen?

With the Meta Infos add-on the meta fields for articles, categories, media and languages can be created and designed.

Meta Infos are intended to manage page or media specific information, such as search engine keywords, an individual header image, whether it should be considered as a news teaser, whether it should appear in certain navigations, copyright info for certain photos, etc.

> Metainfo fields can be defined for articles, categories as well as media. 

Reviewer 2: Auch hier wieder die andere Schreibweise..
